README.md

```diff
@@ -113,7 +113,7 @@ for (const iconSlug in simpleIcons) {
 There are also TypeScript type definitions for the Node package. To use them, simply run:
 
 ```shell
-npm install @types/simple-icons
+npm install -D @types/simple-icons
```

Comment:
Minor but I think we should go with `--save-dev` for clarity to those that are not as familiar with the npm CLI.

Most people will copy this anyway so it doesn't affect them. If you're going to type it manually than using `-D` or `--save-dev` is up to you so it doesn't matter to much what this says.

```suggestion
npm install --save-dev @types/simple-icons
```

Reply:
I agree!